### ISSUE TITLE
Study pad opened in wrong pane due to window activation during sliding main menu open

### ISSUE THREAD

**Issue by Noitarud** [AndBible/and-bible#2868]
This is true of the backup mentioned in #2808 (true of beta). Steps:
1. Select pane you want to switch to a study pad view. Maybe third or fourth to give it more opportunities to misbehave.
2. Slide open left menu and choose study pad.
3. Change workspace and do it again if you need to…

It can open in first or second or third, and not even to the same one if you repeat the steps on same workspace.

**Comment by tuomas2** [comment]
Can't reproduce, can you record a screencast to make sure I understand your issue?

**Comment by Noitarud** [comment]
I've just figured it out. When I do the slide-across to open the left menu the app also registers a tap on the pane and changes the pane. This is easily reproduced. I am using a tablet, android 10.

**Comment by Noitarud** [comment]
I just noticed a related problem. Double tapping makes it go full screen, but if the double tap happens to be on a link of some kind (eg strongs number) it will respond to that too.

**Comment by tuomas2** [comment]
> I've just figured it out. When I do the slide-across to open the left menu the app also registers a tap on the pane and changes the pane. This is easily reproduced. I am using a tablet, android 10.

I don't quite get this either. Video about this?

**Comment by tuomas2** [comment]
> I just noticed a related problem. Double tapping makes it go full screen, but if the double tap happens to be on a link of some kind (eg strongs number) it will respond to that too.

This is known behavior. If you have suggestion about how it should work feel free to create another ticket.

**Comment by Noitarud** [comment]
I did it since you insisted, but both problems are as a result of the same defect.

In the video I swiped across from the left wall to open the left side menu, I did this five times, swiping across from the left wall of each pane.

Video sent to your email since GH has failed me.

**Comment by tuomas2** [comment]
https://github.com/AndBible/and-bible/assets/5811789/d632fc01-aab2-481c-9442-46dd3b23d347

**Comment by Noitarud** [comment]
Another: In landscape view, if divider is close to left wall, and I attempt to move it away, the left menu opens. After closing it the divider is still yellow. Trying to outrun does not work. This one can be worked-around.
As can be seen, the active pane did not change! So the solution for the above would be to ignore taps there?
(Image here!)
![Screenshot_20231012-132458](https://github.com/AndBible/and-bible/assets/98794719/84414bf3-d952-4f11-b9f5-239c4f901133)

**Comment by tuomas2** [comment]
> Another: In landscape view, if divider is close to left wall, and I attempt to move it away, the left menu opens. After closing it the divider is still yellow. Trying to outrun does not work. This one can be worked-around. As can be seen, the active pane did not change! So the solution for the above would be to ignore taps there? 

Workaround is to create window and move it to be the first window. then resizing can be done. Could create another issue about this.

**Comment by Noitarud** [comment]
Yeah I was thinking it really is another, this is something foreground taking focus from workspace, the ones I started out with was the workspace responding as well to action intended for (invisible) foreground only. But. It would be acceptable to have this bug as a solution for the yellow bar issue - if the bar still moved even with the left wall menu opened, at least it was moved away. Will create…

**Comment by tuomas2** [comment]
Can't reproduce this issue on my mobile phone, neither on emulator. :-/

**Comment by tuomas2** [comment]
Okay I see the issue now. Window is being activated because window registers tap same time as slide action happens.

**Comment by Noitarud** [comment]
I have seen the same issue in android itself so I guess you shouldn't elevate it.

Android 12 hides the lll [ ] < keys on split window (app pair) so a flick guesture on either side will also choose active app (horizontal) or is unavailable for top (vertical)
